src/test/suite/extension.test.ts

```diff
@@ -646,4 +671,8 @@ async function getOperatorContainerItems(parentOperator: OperatorItem): Promise<
         }
     }
 	return operatorContainerItems;
+}
+
+async function sleep(ms: number) {
```

Comment:
There's actually already a `sleep` function that exists in the `helper.ts` file. We should use that one instead